src/main/kotlin/no/nav/pam/xmlstilling/hrxml/StillingMapper.kt

```diff
@@ -13,20 +13,22 @@ object StillingMapper {
     private val log = KotlinLogging.logger { }
 
     fun toStillingDto(hrXmlValues: Map<HrXmlStilingParser.HrXmlValue, String>, mottatt: LocalDateTime, eksternBrukerRef: String, arenaId: Int?): XmlStillingDto {
+        val _soknadsfrist = getLocalDate(hrXmlValues.get(SOKNADSFRIST))
+        val _sistePubliseringsdato = getLocalDate(hrXmlValues.get(SISTE_PUBLISERINGSDATO))
         return XmlStillingDto(
                 arbeidsgiver = hrXmlValues.getValue(ARBEIDSGIVER),
                 eksternBrukerRef = eksternBrukerRef,
                 arbeidsgiverBedriftspresentasjon = hrXmlValues.getValue(ARBEIDSGIVER_BEDRIFTSPRESENTASJON),
                 stillingsbeskrivelse = hrXmlValues.getValue(STILLINGSBESKRIVELSE),
                 stillingstittel = hrXmlValues.getValue(STILLINGSTITTEL),
-                soknadsfrist = getLocalDate(hrXmlValues.get(SOKNADSFRIST)),
+                soknadsfrist = if (_soknadsfrist != null) _soknadsfrist else _sistePubliseringsdato,
```

Comment:
Litt mer kotlinish med elvis operator: _soknadsfrist ?: _sistePubliseringsdato

https://kotlinlang.org/docs/reference/null-safety.html

Reply:
En klar forbedring! Jeg forventer at Jenkins nå melder tilbake at "Elvis has entered the build(ing)".